client/app/bundles/course/statistics/pages/StatisticsIndex/students/StudentsStatisticsTable.jsx

```diff
@@ -0,0 +1,259 @@
+                s.groupManagers.map((m) => m.name),
+              ),
+            ),
+          ],
+          logic: (managers, filters) => {
+            if (filters) {
+              const filterSet = new Set(filters);
+              return !managers
+                .map((m) => m.name)
+                .some((name) => filterSet.has(name));
+            }
+            return false;
+          },
+          fullWidth: true,
+        },
+        customFilterListOptions: {
+          render: (name) =>
+            intl.formatMessage(translations.tutorFilter, { name }),
+        },
+        sort: true,
+        customBodyRenderLite: (dataIndex) => {
+          const groupManagers = filteredStudents[dataIndex].groupManagers;
+          if (!groupManagers) {
+            return '';
+          }
+          return (
+            <>
+              {groupManagers.map((m, index) => (
+                <span key={m.id}>
+                  <Link href={m.nameLink} opensInNewTab>
+                    {m.name}
+                  </Link>
+                  {index < groupManagers.length - 1 && ', '}
+                </span>
+              ))}
+            </>
+          );
+        },
+      },
+    });
+  }
+  if (isCourseGamified) {
+    columns.push({
+      name: 'level',
+      label: intl.formatMessage(translations.level),
+      options: {
+        filter: true,
+        sort: true,
+      },
+    });
+    columns.push({
+      name: 'experiencePoints',
+      label: intl.formatMessage(translations.experiencePoints),
+      options: {
+        filter: false,
+        sort: true,
+        alignCenter: true,
+        customBodyRenderLite: (dataIndex) => {
+          const student = filteredStudents[dataIndex];
+          return (
+            <Link
+              key={student.id}
+              href={student.experiencePointsLink}
+              opensInNewTab
+            >
+              {student.experiencePoints}
+            </Link>
+          );
+        },
+      },
+    });
+  }
+  if (showVideo) {
+    columns.push({
+      name: 'videoSubmissionCount',
+      label: intl.formatMessage(translations.videoSubmissionCount, {
+        courseVideoCount,
+      }),
+      options: {
+        alignCenter: true,
+        customBodyRenderLite: (dataIndex) => {
+          const student = filteredStudents[dataIndex];
+          return (
+            <Link
+              key={student.id}
+              href={student.videoSubmissionLink}
+              opensInNewTab
+            >
+              {student.videoSubmissionCount}
+            </Link>
+          );
+        },
+      },
+    });
+    columns.push({
+      name: 'videoPercentWatched',
+      label: intl.formatMessage(translations.videoPercentWatched),
+      options: {
+        filter: false,
+        sort: true,
+        alignCenter: true,
+        customBodyRender: (value) => (
+          <Box sx={{ width: '100%' }}>
+            <LinearProgressWithLabel value={value} />
```

Comment:
Maybe consider allowing `LinearProgressWithLabel` to expose the `className` or `sx` of its outermost `Box`? Then we need not re-wrap with another `Box` here and instead have:
```jsx
<LinearProgressWithLabel sx={{ width: '100%' }} value={value} />
```

Will be sweet if the whole thing doesn't even use `Box`, though. But looks like this component was already written this way, so 🤷.

Reply:
Dont think we need the Box 100% anyway so I will remove it for now..